.gitignore

```diff
@@ -22,6 +22,7 @@
 
 # Build products
 /build*/
+modules/DispModule/sgbm_cuda/build/
```

Comment:
This addition shouldn't be necessary as the previous line covers subdirectories as well.
Why you pushed the build once is another story.